requirements/test.txt

```diff
@@ -1,5 +1,3 @@
--r base.txt
--r cext.txt
```

Comment:
Stops our tests from unconditionally installing things that might (or should) be found in `kolibri/dist`